cypress/integration/ampSpec.js

```diff
@@ -42,6 +42,17 @@ describe('AMP Tests on a .amp page', () => {
     ampScript.should('have.attr', 'src', 'https://cdn.ampproject.org/v0.js');
   });
 
+  it('should contain the AMP boilerplate', () => {
+    // .eq(1) gets the amp <script> as the first loaded is a Cypress <script>
+    const ampBoilerplateStyled = getElement('head style').eq(2);
+    ampBoilerplateStyled.should('have.attr', 'amp-boilerplate');
+  });
+
+  it('should contain the AMP boilerplate noscript version', () => {
+    const ampBoilerplateNoscript = getElement('head noscript');
+    ampBoilerplateNoscript.should('contain', 'amp-boilerplate');
```

Comment:
Instead I'd have `getElement('head noscript style')` and then do `have.attr` as above (rather than merely `contain`)

Reply:
This doesn't seem possible: `noscript` acts as though it contains only a String. Cypress receives this on `getElement('head noscript')`, & fails when. attempting to make it match the suggestion.

<img width="549" alt="screen shot 2018-11-15 at 16 33 06" src="https://user-images.githubusercontent.com/11646957/48567464-faab6180-e8f4-11e8-866b-41bde48aec1c.png">

Reply:
[It doesn't seem there is currently a way to test URLs with JavaScript disabled in Cypress](https://github.com/cypress-io/cypress/issues/1611), either. 😔